### DISCUSSION TITLE
如何让 Radio 跟文字上对齐

### DISCUSSION THREAD

**Discussion by crazyair** [ant-design/ant-design#52033] [General]
https://codesandbox.io/p/sandbox/radio-group-chui-zhi-antd-5-22-5-forked-6m4339?file=%2Fdemo.tsx%3A4%2C20

![image](https://github.com/user-attachments/assets/4f614e28-cc01-4fc7-a946-86cb33871892)

**Comment by petercat-assistant** [comment]
要让 Ant Design 的 Radio 组件与文字上对齐，您可以通过调整 CSS 样式来实现。以下是一个简单的解决方案：

```css
.ant-radio-wrapper {
  display: flex;
  align-items: center;
}
```

在您的代码中，您可以在包含 Radio 组件的容器上应用此样式。这样可以确保 Radio 按钮与文字在垂直方向上居中对齐。

如果您在代码中使用的是内联样式或 CSS-in-JS 解决方案，您可以将 `alignItems: 'center'` 添加到相应的样式对象中。

请根据您的项目结构和样式管理方式选择合适的实现方法。

如果您有其他问题或需要进一步的帮助，请随时告诉我！

<details>
<summary>🪧 Tips</summary>
For further assistance, please describe your question in the comments and @petercat-assistant to start a conversation with me.
</details>
